src/App.js

```diff
@@ -1,17 +1,29 @@
 import React from 'react';
 import './App.css';
+import NewTodo from './components/NewTodo/NewTodo';
 
 import users from './api/users';
+import todos from './api/todos';
+
+const usersList = users.map(user => ({
+  id: user.id,
+  name: user.name,
+}));
+
+const todosList = todos.map(todo => ({
+  ...todo,
+  name: users.find(user => user.id === todo.userId).name,
+}));
```

Comment:
Це можна перенести всередину в функцію, і нічого не зміниться.
```suggestion
function App() {

  const usersList = users.map(user => ({
    id: user.id,
    name: user.name,
  }));
  
  const todosList = todos.map(todo => ({
    ...todo,
    name: users.find(user => user.id === todo.userId).name,
  }));
```

Reply:
рекомендую все таки не держать это в компоненте, все сделано правлиьно